experiments/calibration/PBS_calibration.pbs

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#PBS -N derecho_calibration
+#PBS -o output.txt
+#PBS -e error.txt
+#PBS -l walltime=12:00:00
+#PBS -l select=1:ncpus=4:ngpus=1
+
+## Account number for CliMA
+#PBS -A UCIT0011
+#PBS -q main
+
+module load climacommon
```

Comment:
This will not work in general if you don't also add an export for the MODULEPATH